http/src/lib.rs

```diff
@@ -407,7 +420,11 @@ fn serve<M: jsonrpc::Metadata, S: jsonrpc::Middleware<M>>(
 		bind_result.and_then(move |(listener, local_addr)| {
 			let allowed_hosts = server_utils::hosts::update(allowed_hosts, &local_addr);
 
-			let http = server::Http::new();
+			let http = {
```

Comment:
sorry, didn't check the docs
strange variant for using in api from hyper guys though :)